lib/remote_system.rb

```diff
@@ -140,13 +158,15 @@ def read_file(file, options = {})
       stdout: :capture,
       privileged: options.fetch(:privileged, false)
     }
-    run_command("cat", file, command_options)
-  rescue Cheetah::ExecutionFailed => e
-    if e.status.exitstatus == 1
-      # File not found, return nil
-      return
-    else
-      raise
+    begin
```

Comment:
The `begin ... end`  should not be necessary. Is there a reason why you added it here?